conda/meta.yaml

```diff
@@ -7,24 +7,27 @@ source:
     git_rev: v1.6.0
 
 build:
-    script: python setup.py install
+    number: 1
```

Comment:
Add the build number so we can increase it to 1 (since you already have a build of 1.6.0 on the `e3sm` channel).